anitya/__init__.py

```diff
@@ -145,7 +145,8 @@ def log(session, project=None, distro=None, topic=None, message=None):
                              'project: %(project)s',
         'project.edit': '%(agent)s edited the project: %(project)s fields: '
                         '%(changes)s',
-
+        'project.flag': '%(agent)s flagged the project: %(project)s',
```

Comment:
The admin can already see the reason when looking at the list of flags. The reason is listed there. I just excluded it from this line to avoid a large block of text when viewing the *logs*. Do you think the full reason should be listed on the Flags *and* Logs page? If so, I can fix that.